vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/reasoner/util/TermToVLogConverter.java

```diff
@@ -0,0 +1,54 @@
+package org.semanticweb.vlog4j.core.reasoner.util;
+
+/*-
+ * #%L
+ * VLog4j Core Components
+ * %%
+ * Copyright (C) 2018 VLog4j Developers
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import org.semanticweb.vlog4j.core.model.api.Blank;
+import org.semanticweb.vlog4j.core.model.api.Constant;
+import org.semanticweb.vlog4j.core.model.api.TermVisitor;
+import org.semanticweb.vlog4j.core.model.api.Variable;
+
+public class TermToVLogConverter implements TermVisitor<karmaresearch.vlog.Term> {
+
+	private karmaresearch.vlog.Term vlogTerm;
+
+	public karmaresearch.vlog.Term getVlogTerm() {

```

Comment:
This method and the internal term seems to be not needed, since one can use the value returned by accept directly. If we ever have function terms where the visitor would visit recursively, it will be difficult to store one result term internally anyway, since each functional term would trigger many calls to visit methods of the same visitor.

Reply:
I removed them.